cli/js/lib.deno_runtime.d.ts

```diff
@@ -7,12 +7,12 @@
 /// <reference lib="esnext" />
 
 declare namespace Deno {
-  // @url js/os.d.ts
```

Comment:
I don't think those should be removed

Reply:
What purpose does it have?

Reply:
I'm not sure, but they might be hints for source map, if CI goes green without them then I don't care

Reply:
I think it's just cruft from the autogen.